examples/skip_plot_map_in_3d.py

```diff
@@ -39,10 +38,12 @@
 plotter.plot_line(line)
 
 # Define a SkyCoord for to set the positon of the camera
-camera_position = SkyCoord(0*u.deg, 0*u.deg, 8*const.R_sun, obstime=m.observer_coordinate.obstime, frame=frames.HeliographicStonyhurst)
+camera_position = m.observer_coordinate
 plotter.set_camera_coordinates(camera_position)
```

Comment:
Can we not pass in `m.observer_coordinate` directly?

Reply:
@dstansby wanted it this way I think?

Reply:
I don't think I did, it's fine to just put `m.observer_coordinate` inside the method call.

Reply:
Oh my apologies then